src/data/games/1834.json

```diff
@@ -731,7 +731,7 @@
             "companies": [
               {
                 "label": "OST",
-                "color": "blue"
+                "color": "cyan"
```

Comment:
I'll fix the game, but the idea is that you should just do this: `"companies": ["OST"]` is that not working here? That will pull the correct color and other stuff.